instrumentation/github.com/aws/aws-sdk-go-v2/otelaws/attributes.go

```diff
@@ -31,32 +31,45 @@ const (
 	RegionKey    attribute.Key = "aws.region"
 	ServiceKey   attribute.Key = "aws.service"
 	RequestIDKey attribute.Key = "aws.request_id"
+	TableNameKey attribute.Key = "aws.table_name"
```

Comment:
This attribute key doesn't match the specification's requirement: https://github.com/anuraaga/opentelemetry-specification/blob/c38800cdc4b1551e8d64f262c80c1d625c4ebc29/specification/trace/semantic_conventions/instrumentation/aws-sdk.md#dynamodb

I'm not seeing SQS mentioned in there, but I think since other AWS services have semantic conventions described, so should SQS.

Reply:
Agreed. These are pre-existing X-Ray specific attributes that predate the semantic conventions. Ideally, the X-Ray exporter would look for and favour the semantic convention key for Dynamo, but it's currently not doing that. Instead it's looking for `aws.table_name`. This is being populated by the Java auto-instrumentor and likely others, and IMO should be populated in this instrumentation library.

Reply:
The X-Ray exporter should be changed to handle the translation from the OTel semantic conventions properly.  I'd rather not have this instrumentation emitting non-standard attributes, and twice at that since it is already emitting the attribute with the conventional name.

Reply:
@Aneurysm9 ok, fair enough. How about I update this PR to only include the new `SQSAttributeSetter` and set the `semconv.MessagingURLKey` attribute? This way we're include the specific semantic convention key that is currently not being set and separately, I can open a PR for the X-Ray exporter to extract the semantic convention keys instead.

Reply:
@Aneurysm9 / @dmathieu I have updated the PR to remove custom attributes and instead only included applicable attributes from the semantic conventions. Please could you re-review.